docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -43,6 +43,10 @@ To compile and build just the core systems:
 CORE_ONLY=1 ./gradlew build
 ```
 
+{% hint style="info" %}
+If Gradle uses too much/too little CPU, adjust the `workers.max` property in the `gradle.properties` file accordingly.
```

Comment:
We should also mention that you can adjust this inline using `ORG_GRADLE_WORKERS_MAX=3`

Reply:
I've added documentation specifying all 3 approaches. I've also commented out the property in the `gradle.properties` file. This way, a dev can choose whichever approach suits them best. What do you think?

Note: just setting the env var didn't work for me. I had to use the commands per the documentation I just added.